src/flows/pipes/Notification/Endpoints.tsx

```diff
@@ -23,6 +23,7 @@ export const DEFAULT_ENDPOINTS = {
       ['array', 'slack'].map(i => `import "${i}"`).join('\n'),
     generateQuery: data => `task_data
 	|> schema["fieldsAsCols"]()
+      |> set(key: "_notebook_link", value: "${window.location.href}")
 	|> monitor["check"](
```

Comment:
the `|> set` query has to go after `schema["fieldAsCols"]()` because fieldAsCols gets rid of the `_notebook_link` column